src/gui/shell.cpp

```diff
@@ -459,6 +459,10 @@ void Shell::handleRedraw(const QByteArray& name, const QVariantList& opargs)
 		}
 
 		emit neovimTablineUpdate(curtab, tabs);
+	} else if (name == "option_set") {
+		if (opargs.size() == 2) {
```

Comment:
For forward compatibility it's better to check `opargs.size() >= 2`. Additional items will always be "optional" semantically.